jenkins/root-build.cmake

```diff
@@ -696,6 +712,8 @@ function(CONFIGURE_ROOT_OPTIONS)
       set(specflags ${specflags} -DCMAKE_CXX_STANDARD=17)
     elseif("cxx14" IN_LIST SPECLIST)
       set(specflags ${specflags} -DCMAKE_CXX_STANDARD=14)
+    elseif("root7" IN_LIST want_modules)
+      set(specflags ${specflags} -DCMAKE_CXX_STANDARD=14)
```

Comment:
Does this mean that any build but CentOS7 will be built with C++14?

Reply:
Idea that root7 enabled on all platforms where it can be build. Centos will be excluded.
And only when root7 is where, one have to enable c++14. 
And seems to be I have to check, when c++ standard already specified